### PR TITLE
Feature/#103 Speedometer unit options

### PR DESCRIPTION
**Pull request by maxwellpark**
Dropdown to select metres per second, kilometres per hour, or miles per hour

### CODE REVIEW

File: Assets/Scripts/Player/PlayerConstants.cs [hunk 1]
```diff
@@ -92,21 +91,29 @@ public static class PlayerConstants
...
-    public static string ConsoleTooltip = "Toggle Dev Console (`).";
-
+    public static string TutorialTooltip = "Toggle Tutorial UI.";
+    public static string GhostTooltip = "Toggle Ghost Runner.";
+    public static string ConsoleTooltip = "Toggle Dev Console (`).";
+
     public static string ResolutionTooltip = "Screen Resolution.";
     public static string GraphicsTooltip = "Graphics Quality.";
-    public static string FullscreenTooltip = "Toggle Fullscreen.";
+    public static string FullscreenTooltip = "Toggle Fullscreen.";
```
Comment (ambid17) [inline Assets/Scripts/Player/PlayerConstants.cs:107]: what is this lmao, fix your line endings

**Review by ambid17 — CHANGES_REQUESTED** [review]
need to adjust the text box size

File: Assets/Scripts/Player/PlayerConstants.cs (hunk 1 above)
Comment (maxwellpark) [reply Assets/Scripts/Player/PlayerConstants.cs:107]: lines in the sand

Comment (maxwellpark) [reply Assets/Scripts/Player/PlayerConstants.cs:107]: http://gph.is/2af76jd

**Pull request merged**